README.md

```diff
@@ -61,6 +61,23 @@ Consider the following example, which allows changes to certain paths without
 review, but all other changes require review from the `palantir/devtools`.
 Any member of the `palantir` organization can also disapprove changes.
 
+#### Notes on YAML Syntax
```

Comment:
I'd place this after the sample YAML block and before the `Remote Policy Configuration` section. Right now, the previous text says "Consider the following example" but then this section appears before the example.

Reply:
Oops, good catch